src/App.jsx

```diff
@@ -1,15 +1,24 @@
 import "./App.css";
 import { Header } from "./components/Header.jsx";
 import { Footer } from "./components/Footer.jsx";
-import { AllArticles } from "./components/AllArticles.jsx";
+import { ArticlesList } from "./components/ArticlesList.jsx";
 import "bootstrap/dist/css/bootstrap.min.css";
+import { Route, Routes } from "react-router";
+import { Home } from "./components/Home.jsx";
+import { Article } from "./components/Article.jsx";
 
 function App() {
   return (
     <>
       <Header />
-      <AllArticles />
-      <Footer/>
+      <main>
+        <Routes>
+          <Route path={"/"} element={<Home />} />
+          <Route path={"/articles"} element={<ArticlesList />} />
+          <Route path={"/articles/:article_id"} element={<Article />} />
+        </Routes>
+      </main>
+      <Footer />
```

Comment:
I saw your note in the PR about the `Home` component not being ready yet 👍 It's up to you of course but one thing you can do while still developing the app is have the `ArticlesList` component on your `/` route until Home is ready to take over